avalon/pipeline.py

```diff
@@ -878,12 +878,13 @@ def create(name, asset, family, options=None, data=None):
 
     host = registered_host()
 
-    plugins = list()
-    for Plugin in discover(Creator):
-        has_family = family == Plugin.family
+    if isinstance(family, str):
+        Plugins = [P for P in discover(Creator) if family == P.family]
+    else:
+        Plugins = [family]
```

Comment:
This got me wondering for a second what is going on here. Should this maybe be in reverse, something like this:

```python
    if isinstance(family, Creator):
        # Allow passing in a specific Creator to run only that plug-in
        Plugins = [family]
    else:
        Plugins = [P for P in discover(Creator) if family == P.family]
```

Also makes the check more explicit to it being a Creator plug-in.

Reply:
Good point !
Changed in commit ebcb78c
( note that the `family` has now changed to also accept a subclass of `api.Creator`, so it should be using `issubclass` ;) )

Reply:
Well spotted!